Console/UI/Pages/EventsLog.xaml.cs

```diff
@@ -78,22 +80,25 @@ public bool IsTrackingEnabled
             get => _isTrackingEnabled;
             set
             {
-                if (_isTrackingEnabled != value)

```

Comment:
Removing this will allow triggering the same code when the value in fact didn't change (and I do the same check all the time for setters since I observed it was triggered every time otherwise - at least in UWP)... Doesn't cost much to keep it there ;)

Reply:
Yes I thought so, and it's probably a good habit but in this case it's only triggered when the refresh button is clicked.